### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by Lykhosherstyy**
Changes: 

- redesign main menu
- improve UI view

### CODE REVIEW

**Comment by Lykhosherstyy** [comment]
@Viposha , plz check my improvements:)

**Review by Viposha — COMMENTED** [review]
@Lykhosherstyy  add white bg color to post. Do you like?

File: iflask/templates/new_post.html [hunk 1]
```diff
@@ -34,16 +34,16 @@
                 {% endif %}
             </div>
             <div class="form-group" >
-                {{ form.picture.label() }}
-                {{ form.picture(class="form-control-file") }}
+                {{ form.picture.label(class="btn btn-secondary") }}
+                {{ form.picture(class="form-control-file is-hidden") }}
```
Comment (Viposha) [inline iflask/templates/new_post.html]: @Lykhosherstyy  "is hidden" hide name of file when you upload image. and i have no idea my file is choosen or not

Comment (Lykhosherstyy) [reply iflask/templates/new_post.html]: ok, I'll revert it

Comment (Lykhosherstyy) [reply iflask/templates/new_post.html]: @Viposha , fixed it.   https://github.com/Viposha/iflask/pull/4/commits/c4f5cf32989c0c3772442bf3f52e377658b22072

**Pull request merged**